### ISSUE TITLE
Youtube errors should be redirected to the screen reader or displayed in a dialog

### ISSUE THREAD

**Issue by Mohamed00** [manuelcortez/TWBlue#202]
It's possible for Youtube itself to return errors, such as if the uploader of a video has not made the video available in the user's country. Currently these are logged to the console,but I think they should be presented in some user facing way, such as a message through the screen reader or an error dialog.
